Dungeoneer/src/com/interrupt/dungeoneer/gfx/GlRenderer.java

```diff
@@ -417,8 +417,8 @@ public void initHud() {
 		Gdx.app.log("DelverLifeCycle", "Initializing HUD");
 
 		Game.ui.clear();
-		Game.hotbar.init(itemTextures.getSpriteRegions());
-		Game.bag.init(itemTextures.getSpriteRegions());
+		Game.hotbar.init();
```

Comment:
This is a nice catch! But strange we don't actually use the sprite regions.

Reply:
Yeah, it is strange. Maybe a left-over of some base class that never got implemented. Leaving as is for now.